docs/tutorial/herbsself.md

```diff
@@ -2,15 +2,51 @@
 id: herbsshelf
 title: 6. Generating Herbs Shelf
 sidebar_label: 6. Generating Herbs Shelf
-slug: /tutotial/herbsshelf
+slug: /tutorial/herbsshelf
 ---
 
-Introduction to Herbs Shelf
+## Introduction to Herbs Shelf
+
+> Herbs Shelf is a self-generated documentation based on your use cases and entities from your domain.
+>
+> — [Herbs Shelf | HerbsJS](/docs/glues/herbsshelf)
+
+![](../../static/img/herbsshelf_screenshot.png)
 
 ## Herbs Shelf Setup
 
-- Screenshoot
+To setup the automatic documentation, we need a list with all usecases to be included in the documentation. With CLI, it is in `src/domain/usecases/index.js`:
+
+```js
+// src/domain/usecases/index.js
+module.exports = [
+    // the tags are to store metadata, such as the group and type of the use case
+    { usecase: require('./user/createUser'), tags: { group: 'Users', type: 'mutation'} },
+    { usecase: require('./user/updateUser'), tags: { group: 'Users', type: 'mutation'} },
+    { usecase: require('./user/deleteUser'), tags: { group: 'Users', type: 'mutation'} },
+    { usecase: require('./user/findOneUser'), tags: { group: 'Users', type: 'query'} }
+]
+```
+
+Once you have this file, you can use it with `@herbsjs/herbsshelf` to get the HTML content and do what you want with.
```

Comment:
I wouldn´t say "do what you want with", because for now, the Herbs Shelf only renders the HTML content of the usecases inside the route that it´s being set up below.

So it could be "... to render the HTML content and set a route called...."